DefaultCiActionResolver.php

```diff
@@ -59,7 +71,15 @@ public function addPossibleRoutes(CiActionResolveEvent $event, &$pathParts, $ind
     public function onActionResolveEvent(CiActionResolveEvent $event)
     {
         $path = $event->getRequest()->getPathInfo();
-        $parts = explode('/', substr($path, 1));
+        $part = (string) substr($path, 1);
+        if ($part === '') {
+            $part = 'home/';
```

Comment:
As I remember I got a problem with index page, my code igniter didn't return it as a route and symfony will show a 404 page.
I will investigate it again. And will answer directly